src/Microsoft.Performance.SDK.Runtime/Discovery/ReflectionPlugInCatalog.cs

```diff
@@ -28,21 +28,21 @@ namespace Microsoft.Performance.SDK.Runtime.Discovery
     ///     will be set to a non-null value. If <c>false</c> is returned, <paramref name="reference" />
     ///     will be set to <c>null</c>.
     /// </returns>
-    public delegate bool TryCreateCustomDataSourceReferenceDelegate(
+    public delegate bool TryCreateProcessingSourceReferenceDelegate(
         Type type,
-        out CustomDataSourceReference reference);
+        out ProcessingSourceReference reference);
 
     /// <summary>
-    ///     This class registers to with a provider to receive Types that might be custom data sources.
-    ///     The types are evaluated and stored as a <see cref="CustomDataSourceReference"/> when applicable.
+    ///     This class registers to with a provider to receive Types that might be <see cref="IProcessingSource"/>s.
+    ///     The types are evaluated and stored as a <see cref="ProcessingSourceReference"/> when applicable.
     /// </summary>
     public sealed class ReflectionPlugInCatalog
```

Comment:
>PlugIn [](http://example.com/codeflow?start=34&length=6)

same comment on PlugIn vs ProcessingSource.

Reply:
will hopefully be solved with #22, so deferring this for now